cea/optimization/slave/seasonal_storage/design_operation.py

```diff
@@ -381,8 +381,9 @@ def Storage_Design(CSV_NAME, SOLCOL_TYPE, T_storage_old, Q_in_storage_old, locat
              "Q_rejected_fin":Q_rejected_fin,
              "P_HPCharge_max":P_HP_max
             })
-        Name = MS_Var.configKey + "StorageOperationData.csv"
-        os.chdir(locator.get_optimization_slave_results_folder())
+        Name = locator.get_optimization_slave_StorageOperationData(MS_Var.configKey)
```

Comment:
btw: `Name`, `fName` etc. are soooo non-PEP8. Also, they're really bad variable names for any coding convention. I know you inherited this code and I know you need to get your job done, so I'm not expecting you to go fix this, just... don't make the same mistakes in your own code. `Name` is just lazy thinking and lazy thinking makes reading code hard. Instead, just use `storage_operation_data_path`. Woohoo. now you know what it is *and* what it's for!